os/kernel/init/os_bringup.c

```diff
@@ -104,7 +104,7 @@
 #if defined(CONFIG_INIT_ENTRYPOINT)
 /* Initialize by starting a task at an entry point */
 
-#ifndef CONFIG_USER_ENTRYPOINT
+#if !defined(CONFIG_USER_ENTRYPOINT) && !defined(CONFIG_TASH)
```

Comment:
Since tash_start() happens in preapp_start(), would it be better to check for CONFIG_SYSTEM_PREAPP_INIT also?

Reply:
How about removing `SYSTEM_PREAPP_INIT` config? Because there is no way to use TASH without preapp init. But if there is nothing to launch some services, we don't need to make a preapp task. Then, we need it... Do you have any suggestion for that?

Reply:
or making a dependancy between preapp config and tash config. When preapp config is enabled, user can enable tash?? Is it bad?